test/activesupport6/encoding_test.rb

```diff
@@ -74,42 +74,77 @@ def test_hash_keys_encoding
     ActiveSupport.escape_html_entities_in_json = false
   end
 
-  def test_utf8_string_encoded_properly
-    # The original test seems to expect that
-    # ActiveSupport.escape_html_entities_in_json reverts to true even after
-    # being set to false. I haven't been able to figure that out so the value is
-    # set to true, the default, before running the test. This might be wrong but
-    # for now it will have to do.
-    ActiveSupport.escape_html_entities_in_json = true
-    result = ActiveSupport::JSON.encode("€2.99")
-    assert_equal '"€2.99"', result
-    assert_equal(Encoding::UTF_8, result.encoding)
-
-    result = ActiveSupport::JSON.encode("✎☺")
-    assert_equal '"✎☺"', result
-    assert_equal(Encoding::UTF_8, result.encoding)
```

Comment:
The test functions in this file were taken directly from the ActiveSupport code base. I did add the comment and the one line described. Is there a reason to remove the test?

Reply:
I see that it was not removed but moved and the comment deleted. Is the comment no longer valid?

Reply:
I moved it into the `UnicodeTests` module so that it could be included and run twice - once with `escape_html_entities_in_json` on and once with it off - https://github.com/ohler55/oj/pull/912/files/eb3febb2e03389b573ba1144eb301a0629a7de5a#diff-5919a44f303dc6030d533e858a18ac8298c483aa397f31c89cf465e09ef5447aR134

Reply:
> I see that it was not removed but moved and the comment deleted. Is the comment no longer valid?

Yes - the module the test is included in now explicitly sets `escape_html_entities_in_json` (to both values) in a `setup` block. So it passes with that setting both on and off, so I don't think the comment is necessary anymore.

Reply:
Right but the point of the comment was to let someone looking at the test know. what the modifications were to the original ActiveSupport tests. Given that you should probably note the module was set up to set that value which is different than the original tests. I'm happy to have the module there but it does need a comment explaining why it is there.

Reply:
The original activesupport test _also_ runs with both `escape_html_entities_in_json` set to both values: this is running the test cases multiple times with different parameters: https://github.com/rails/rails/blob/56bcc0abd3c9a6b09469e9428f6eea0dd77c2294/activesupport/test/json/encoding_test.rb#L22-L37

so AFAICT this test is now running exactly as intended in the upstream repo and is identifical? I think I _removed_ the point of difference?